resource-managers/yarn/src/main/resources/META-INF/services/org.apache.spark.deploy.security.ServiceCredentialProvider

```diff
@@ -0,0 +1,3 @@
+org.apache.spark.deploy.yarn.security.YARNHadoopFSCredentialProvider
```

Comment:
Ah, yes.  This is a problem.  I'll write the unit test, and move the core META-INF to Mesos.  I was hoping for a general set of registered providers, but I guess each scheduler will need to register their own.